bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -110,6 +110,10 @@ spec:
           - ""
           resources:
           - configmaps
+          - pods
+          - secrets
+          - serviceaccounts
+          - services
```

Comment:
I know this looks scary, but there shouldn't be a real diff here. I just changed the kubebuilder stuff to be grouped together correctly.